src/main/java/school/lemon/changerequest/java/introduction/hw1/Task8.java

```diff
@@ -2,6 +2,10 @@
 
 public class Task8 {
     public static int calculateSum(int N) {
-        return 0;
+
+        int res = 0;
+        for (int i = 0; i <= N; i++)
+            res += (N + i) * (N + i);
```

Comment:
You don't have to invent `pow` function. Please think about possible future changes in the requirements, what if the formula would be changed, so that you'd have to calculate (N=i)^15, will you add additional braces? - It's almost always better to use functions from Java standard library.